cmd/juju/status/formatter.go

```diff
@@ -76,6 +76,11 @@ func newStatusFormatter(p newStatusFormatterParams) *statusFormatter {
 	return &sf
 }
 
+// Format formats the model status.
+func (sf *statusFormatter) Format() (formattedStatus, error) {
+	return sf.format()
+}
+
 func (sf *statusFormatter) format() (formattedStatus, error) {
```

Comment:
This is just used in 1 place, so can just export directly.

Reply:
done.